assets/templates/accounts/register.html

```diff
@@ -31,6 +31,10 @@ <h3>Personal information</h3>
                         <h3>Memberships</h3>
                         {% include "snippets/bootstrap_form.html" with form=associationlink_form %}
 
+                        <br>
+                        <span class="text-danger">*</span> required<br>
```

Comment:
Use `<p></p>` instead of `<br>`, standalone text should almost always be put inside `p` tags, not doing so is considered bad practice.